src/sysadmin/user-impersonation.md

```diff
@@ -53,9 +53,14 @@ Follow these steps to use the user impersonation feature:
 
 The user impersonation feature configuration options.
 
-* `switch_user_feature.enabled` (Enable or disable the feature, default: `false`)
-* `switch_user_allow_listed_ips` (Default IP(s) allowed to use the feature,
-  default: `localhost,127.0.0.1,[0:0:0:0:0:0:0:1]`)
+* `switch_user_feature.enabled` (Enable or disable the feature, default: `disabled`)
+* `switch_user_allow_listed_ips` (Default allowed IP(s) are; `localhost,127.0.0.1,[0:0:0:0:0:0:0:1]`)
+
+## Security restrictions
+* Feature must be enabled in the `dhis.conf` configuration file, default value is; `disabled`.
+* Users trying to imperonate needs to send requests from an allowed IP.
```

Comment:
there's a few typos and grammar changes needed on this line, corrected looks like this:
`Users trying to impersonate need to send requests from an allowed IP.`